core/src/main/java/de/danoeh/antennapod/core/mapper/CursorMapper.java

```diff
@@ -0,0 +1,226 @@
+package de.danoeh.antennapod.core.mapper;
+
+import android.database.Cursor;
+
+import java.util.Date;
+
+import de.danoeh.antennapod.core.feed.Chapter;
+import de.danoeh.antennapod.core.feed.Feed;
+import de.danoeh.antennapod.core.feed.FeedFilter;
+import de.danoeh.antennapod.core.feed.FeedItem;
+import de.danoeh.antennapod.core.feed.FeedMedia;
+import de.danoeh.antennapod.core.feed.FeedPreferences;
+import de.danoeh.antennapod.core.feed.ID3Chapter;
+import de.danoeh.antennapod.core.feed.SimpleChapter;
+import de.danoeh.antennapod.core.feed.VolumeAdaptionSetting;
+import de.danoeh.antennapod.core.feed.VorbisCommentChapter;
+import de.danoeh.antennapod.core.storage.PodDBAdapter;
+import de.danoeh.antennapod.core.util.SortOrder;
+
+public class CursorMapper {
```

Comment:
This file is already getting pretty large. Could you please create one mapper for each class? Also, I think that writing to the cursor should probably also be handled here. That reduces the number of files that need to be touched when something in the database changes.

Reply:
Agree.